code/modules/mob/living/carbon/human/whisper.dm

```diff
@@ -109,24 +109,18 @@
 			message = replacetext(message, "u", "�")
 			message = replacetext(message, "b", "�")
 
-	var/list/listening = hearers(message_range, src)
+	var/list/listening = get_mobs_or_objects_in_view(message_range, src)

```

Comment:
I'm cloning Bay, so give me a minute to read through it in a less ugly interface than GitHub. I may be back with questions. Or if it's super-awesome I may just close this and redo it with your suggestion.